mypy.ini

```diff
@@ -0,0 +1,9 @@
+[mypy]
```

Comment:
let's add an issue for this https://github.com/python/mypy/issues/5205

Reply:
yep 👍 Of course if we had a setup.cfg you could also add it there. It will be nice when setuptools moves their config to the toml, then we can do away with these horribly bespoke setup.json files. I actually had a little look at adding setup.cfg to aiida registry, but its a bit of a pain reading these cfg files easily (although possible).